src/Metadata/Resource/Factory/XmlResourceMetadataFactory.php

```diff
@@ -0,0 +1,206 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ApiPlatform\Core\Metadata\Resource\Factory;
+
+use ApiPlatform\Core\Annotation\Resource;
+use ApiPlatform\Core\Exception\ResourceClassNotFoundException;
+use ApiPlatform\Core\Metadata\Resource\ResourceMetadata;
+use DOMDocument;
```

Comment:
According to Symfony CS, we don't add use statements for classes of the global namespace (we just prefix them with `\`)